packages/nimble-components/src/card/styles.ts

```diff
@@ -1,35 +1,34 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '@microsoft/fast-foundation';
 import {
+    bodyEmphasizedFontWeight,
     bodyFont,
     bodyFontColor,
-    borderColor,
-    borderWidth,
-    sectionBackgroundColor,
-    standardPadding,
-    titleFont,
-    titleFontColor
+    cardBorderColor,
+    subtitleFontColor,
+    subtitlePlus1Font
 } from '../theme-provider/design-tokens';
 
 export const styles = css`
     ${display('flex')}
 
     :host {
         flex-direction: column;
-        gap: ${standardPadding};
-        padding: ${standardPadding};
-        border: ${borderWidth} solid ${borderColor};
+        gap: 24px;
+        padding: 24px;
+        border: 2px solid ${cardBorderColor};
+        border-radius: 8px;
         font: ${bodyFont};
         color: ${bodyFontColor};
-        background-color: ${sectionBackgroundColor};
     }
 
     section {
         display: contents;
     }
 
     slot[name='title'] {
-        font: ${titleFont};
-        color: ${titleFontColor};
+        font: ${subtitlePlus1Font};
```

Comment:
We may need to follow up with Brandon on this. Regardless of what options we might give in the future, I'm not sure whether wrapping or truncating on one line is a better default. I'd lean towards letting it wrap by default, but only because we don't currently have a (non-incubating) tooltip solution to show the full text if it's truncated.

Reply:
I checked with Brandon, and he said basically the same thing as Katie:

>I believe the suggestion was that we wrap, but at a max of two lines

I agree with your assessment that there isn't a great way to enforce that with CSS, the two strategies we've used in nimble are:
1. Limit to one line & add ellipsis + tooltip if it's too long. To accomplish this, we actually use the `title` attribute rather than the `nimble-tooltip`. We have an `overflow` helper that helps add the `title` attribute only when the text will overflow. You can take a look at #1669 for an example of where we did that for list options.
2. Document the guidance of how long the text should be. For example, [on the banner story](https://nimble.ni.dev/storybook/?path=/docs/components-banner--docs), we say that the text _should be short enough to fit within three lines when displayed_. This option is impossible for nimble to enforce & can also be tricky for clients if the element is used in a place that may change width based on window width (or some other layout change), but in some cases it's the best we can do because it doesn't make sense to limit text to a single line.

It sounds like maybe option 2 would be the most aligned with the designer's guidance. It's also the easiest to implement (i.e. only adding a sentence to the docs), so if we change our minds later, we haven't done any extra work. That being said, I don't have an issue with option 1 if you think that would be preferred for this component.

Reply:
Thanks for the information! I'm happy with option 2. I'll add a sentence to the docs.

Reply:
Here's what I added to the docs. Let me know if you want something different.
![image](https://github.com/ni/nimble/assets/35350751/94b742b7-7373-41b9-aa96-739cd7f830c8)